flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing even more since the commit say "support only 64-bits". I hate to break it to you but aarch64 IS 64-bits.

Reply:
The commit indeed should have been more descriptive in saying x86_x64 in the commit but this kinda goes off-topic in my opinion.

Also while it is sad that it doesn't support aarch64 it is as linked in the commit message a planned thing for the future.